ssg/constants.py

```diff
@@ -75,6 +75,32 @@
     "Verify Ansible meets SCAP-Security-Guide version requirements."
 standard_profiles = ['standard', 'pci-dss', 'desktop', 'server']
 
+
+OVAL_SUB_NS = dict(
```

Comment:
Does this mean that we are going to have to manually handle the OVAL sub namespaces? If so, can this be automated to would mean it is one less thing to worry about in having to update or maintain.

Reply:
This PR doesn't introduce any new obligation - it just takes what's here and reshuffles it. If another general namespace is introduced, it may be aliased as e.g. `ns6` in the end-products instead of being referred to by nice names.
In case that another OVAL sub-namespace is introduced and we would like to support it in our content, then I think more manual work will be required, and updating the dictionary will be even one of the things needed, and it will be needed for another reasons than to make beautiful datastreams.

Reply:
I don't see new name spaces added so frequently that it warrants automation.